spec/whois.md

```diff
@@ -1,90 +1,33 @@
-- [An ACDC Verifiable Presentation].
+The following is a use case for this capability. Consider an example of the `did:webs` controller being an educational institution that issues "degree" verifiable credentials to its graduate students. A graduate from the school submits as part of their job application to a company a [[ref: Verifiable Presentation]] derived from the verifiable credential they received from the school. The company receiving the presentation can verify the cryptography of the presentation, but can they trust the school that issued the verifiable credential? If the school issued the verifiable credential using its `did:webs` DID, the company can resolve the DID. It can also resolve the DID's `whois` DID URL where it might find VCs from issuers it trusts with the `did:webs` DID as the subject. For example:
 
-[Data Integrity Proof Verifiable Presentation]: https://www.w3.org/TR/vc-data-model/#data-integrity-proofs
-[JSON Web Token Verifiable Presentation]: https://www.w3.org/TR/vc-data-model/#example-verifiable-presentation-using-jwt-compact-serialization-non-normative
-[An ACDC Verifiable Presentation]: https://trustoverip.github.io/tswg-acdc-specification/draft-ssmith-acdc.html
+* A verifiable credential issued by the Legal Entity Registrar for the jurisdiction in which the school is headquartered.
+  * Since the company knows about the Legal Entity Registrar, they can automate this lookup to get more information about the school from the verifiable credential -- its legal name, when it was registered, contact information, etc.
+* A verifiable credential issued by the "Association of Colleges and Universities" for the jurisdiction of the school.
+  * Perhaps the company does not know about the Association for that jurisdiction. The company can repeat the `whois` resolution process for the issuer of _that_ credential. The company might (for example), resolve and verify the `did:webs` DID for the Association, and then resolve the `whois` DID URL to find a verifiable credential issued by the government of the jurisdiction. The company recognizes and trusts that government's authority, and so can decide to recognize and trust the Association.
+* A verifiable credential issued by US News magazine about the school's placement on th [US News Rankings of Colleges and Universities].
 
-When the verifiable presentation is requested, the resolved object MUST
-contain the appropriate MIME type of the verifiable presentation format.
+Such checks can all be done with a handful of HTTPS requests and the processing of the DIDs and verifiable presentations. The result is an efficient, verifiable, credential-based, decentralized, multi-domain trust registry.
 
-It is up to the DID Controller to decide to publish a verifiable presentation
-and if so, which Verifiable Credentials to put into the verifiable presentation.
+[US News Rankings of Colleges and Universities](https://www.usnews.com/education/best-global-universities)
```

Comment:
Is this link supposed to be on line 29?

Reply:
[US News Rankings of Colleges and Universities]: https://www.usnews.com/education/best-global-universities